duckduckgo/Android
# Android: streamline home screen load

Task/Issue URL: https://app.asana.com/0/488551667048375/1200024092221607/f
Tech Design URL: 
CC: 

**Description**:
This PR includes changes to make our clear data restart process smoother and faster:
- Removed fade-in transitions
- DDG logo fixed position
- Removed Data clear message when the user clears data manually (automatic data clear still shows the confirmation message)
- layout optimizations
- dagger graph initialized only for the app process (fire process will not initialize dagger injector)
- Browser activity has a dummy omnibar to avoid showing a white full screen

**Steps to test this PR**:
Test 1:
1. Perform fresh install
1. Clear data in the 3 different possible scenarios (during onboarding, onboarding completed - widget cta, onboarding completed - no Ctas)

Test 2:
1. Ensure automatic data clear works as usual (includes optimization changes but data clear message shows up if app cleared in foreground)
 
> See videos posted in [product review](https://app.asana.com/0/1142021229838617/1200036139228532/f) to validate expected behavior


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
